src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -68,7 +68,7 @@ export const styles = StyleSheet.create({
   countUser: (from: string) => ({
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(16.94),
-    color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+    color: from === SIGNED ? COLORS.signed_primary : COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change seems fine as long as the new color values in `COLORS` match the intended design. Make sure to test these changes thoroughly.
```diff
- color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+ color: from === SIGNED ? COLORS.signed_primary : COLORS.anon_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->